src/JestExt.ts

```diff
@@ -562,7 +457,13 @@ export class JestExt {
       }
     })
 
-    vscode.debug.startDebugging(vscode.workspace.workspaceFolders[0], configuration)
+    const workspaceFolder = vscode.workspace.workspaceFolders[0]
+    try {
+      await vscode.debug.startDebugging(workspaceFolder, 'vscode-jest-tests')
+    } catch {
+      const debugConfiguration = this.debugConfigurationProvider.provideDebugConfigurations(workspaceFolder)[0]
+      vscode.debug.startDebugging(workspaceFolder, debugConfiguration)

```

Comment:
maybe we can capture error here as well and provides some helpful notes to improve the user experience...

Reply:
Unfortunately the exception throwing behavior of `startDebugging` isn't documented. The only case in which it throws (a `null` exception) seems to be if the requested debug configuration isn't present. If starting the debug session fails because of a wrong command in the configuration, vscode displays a more or less detailed error message by itself. The only modification I made was to show a warning message, if it falls back to the default configuration.